test/CMakeLists.txt

```diff
@@ -73,6 +73,8 @@ endif()
 
 find_package(Threads)
 
+find_package(storage_client)
```

Comment:
You may want to add that to the CMakeLists of `xtensor-io` instead, following this [pattern](https://github.com/xtensor-stack/xtensor-io/blob/bab241abb3de8bad7c53f866208d68ebd7affb6d/CMakeLists.txt#L113). This way, the link is automagic for projects using `xtensor-io` if the package is found.